IntoHistory/MainView/MainViewController.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  MainViewController.swift
+//  IntoHistory
+//
+//  Created by Yu ahyeon on 2022/08/10.
+//
+
+import UIKit
+
+class MainViewController: UIViewController {
+
+    // MARK: - Property
+
+    private let scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        scrollView.showsVerticalScrollIndicator = false
+        return scrollView
+    }()
+
+    private let contentView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var flagAreaImage: UIImageView = {
```

Comment:
p1;

접근제어자 추가 부탁드립니다! 
```suggestion
    private lazy var flagAreaImage: UIImageView = {
```

Reply:
넵! 수정하도록 하겠습니다.
혹시 접근제어자를 사용하면 좋은 점 알 수 있을까요?

Reply:
접근제어자는 모듈간의 접근을 제한할 수 있는 기능입니다. 현재 저희 앱처럼 간단한 동작으로 이루어져있는 앱일경우에는 문제되지 않을 수 있지만, 앱의 규모가 커질 때에는 불필요한 접근이 허용되는 상황이 발생하여 의도와는 다른 결과를 초래할 수 있습니다.

그래서 외부에서 호출될 가능성이 없고, 내부에서만 사용될 변수들에 대해서는 private를 무조건 써주는게 안정성 측면에서 훨씬 유리합니👍

Reply:
넵 답변 감사합니다! 반영했습니다!
17b65ba76c2de87fe9b7327b86c8bc5033cf8a08